============================= test session starts =============================
platform win32 -- Python 3.6.2, pytest-3.5.1, py-1.5.3, pluggy-0.6.0
rootdir: C:\Users\rasmusm\Documents\cse-prac7-ci, inifile:
collected 1 item

test_sample.py F                                                         [100%]

================================== FAILURES ===================================
_________________________________ test_answer _________________________________

    def test_answer():
>       assert inc(3) == 5
E       assert 4 == 5
E        +  where 4 = inc(3)

test_sample.py:5: AssertionError
========================== 1 failed in 0.06 seconds ===========================




============================= test session starts =============================
platform win32 -- Python 3.6.2, pytest-3.5.1, py-1.5.3, pluggy-0.6.0
rootdir: C:\Users\rasmusm\Documents\cse-prac7-ci, inifile:
collected 1 item

test_sample.py .                                                         [100%]

========================== 1 passed in 0.03 seconds ===========================
